cpp/src/arrow/buffer.h

```diff
@@ -101,6 +101,14 @@ class ARROW_EXPORT Buffer {
   Status Copy(const int64_t start, const int64_t nbytes,
               std::shared_ptr<Buffer>* out) const;
 
+  /// Zero bytes between padding, i.e. bytes between size_ and capacity_.
```

Comment:
Not a native English speaker, but "between padding" looks grammatically incorrect.

Reply:
"in" instead of "between" sounds fine

Reply:
Yeah, comment editing fail, I'll fix it.